lib/google_drive/spreadsheet.rb

```diff
@@ -131,5 +131,18 @@ def batch_update(requests)
         @session.sheets_service.batch_update_spreadsheet(id, batch_request)
       batch_response.replies
     end
+
+    # Append values to a spreadsheet by first searching for a data table at a range,
```

Comment:
Can you document the type and the format of `range_name` and `values`, maybe with an example code? How does it determine the worksheet which the value is added to?

Can you also document possible values in `override_params`, maybe as a link to the Google's API doc?